docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,301 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    project = mlrun.get_or_create_project("app-demo-flask")
</pre><p>let's use the same project for both functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>